src/server/components/dashboard/__tests__/controllers.test.ts

```diff
@@ -994,11 +1030,4 @@ describe("Dashboard Controllers", () => {
     });
   });
 
-  function mockNextFunction(expectedStatus: number, expectedMessage: string): NextFunction {
```

Comment:
fwiw - abstraction is bad form in unit tests. UT should have as few dependencies as possible.

Reply:
Possible engineering roundtable talk 🤷🏿

Reply:
I can do. Essentially, there is very little benefit of abstracting 3 (or more depending on your linter....) lines of code in test files. It adds additional complexity to unit tests. I have to scroll/navigate back to `next` to figure out what it's doing, and keep that in mind, whilst trying to test for more complex code